letta/server/rest_api/routers/v1/organizations.py

```diff
@@ -50,10 +49,10 @@ def delete_org(
 ):
     # TODO make a soft deletion, instead of a hard deletion
     try:
```

Comment:
you can just use `Organization.default(db_session)` here or do that one layer down in sever (whereever you land the db_session)